doc/new_election/by_election_data_coding_tasks.md

```diff
@@ -4,23 +4,24 @@ In general [db/fixtures/README](../../db/fixtures/README.md) can be a guide. If
 
 ## Preparing data
 
-### Switch to by election mode if need be
-
-See [db/seeds.rb](../../db/seeds.rb) and [db/seeds_for_general_election.rb](../../db/seeds_for_general_election.rb) and edit.
+### YAML files
 
-### YAML file
+A yaml file needs to be prepared with all the party and constituency data. Previous yaml files should be a guide, for example see the commits for the By Election in 2020 and the files
 
-A yaml file needs to be prepared with all the party and constituency data. Previous yaml files should be a guide, for example see the commits for the By Election in 2020 and the files 
+    db/fixtures/be2022.yml
 
     db/seeds.rb
     db/fixtures/be2022/candidate.rb
     db/fixtures/be2022/party.rb
+    db/fixtures/be2022_yaml.rb
     spec/db/fixtures/be2022/candidate_spec.rb
     spec/db/fixtures/be2022/party_spec.rb
-    db/fixtures/be2022.yml
-    db/fixtures/be2022_yaml.rb
     spec/db/fixtures/be2022_yaml_spec.rb
 
+### Switch to by election mode if need be
+
+See [db/seeds.rb](../../db/seeds.rb) and optionally edit so that the default election type is by election. This can be overridden using an environment variable (ELECTION_TYPE=by), however, making the edit means other developers don't need to remember to use the envirobment variables.
```

Comment:
```suggestion
See [`db/seeds.rb`](../../db/seeds.rb) and optionally edit so that the default election type is by election. This can be overridden using an environment variable (`ELECTION_TYPE=by`), however, making the edit means other developers don't need to remember to use the environment variables.
```

Reply:
Fixed in #784.